Python/Typing Speed Test/typingTest.py

```diff
@@ -0,0 +1,164 @@
+import pygame

```

Comment:
You can also comment little bit about work of package you used in the script, that will help to understand the package work.